daemon/src/cfd_actors.rs

```diff
@@ -89,6 +90,19 @@ pub async fn load_cfd(order_id: OrderId, conn: &mut PoolConnection<Sqlite>) -> R
     Ok(cfd)
 }
 
+pub async fn apply_event(
+    process_manager: &xtra::Address<process_manager::Actor>,
+    event: Event,
+) -> Result<()> {
+    if let Err(e) = process_manager
+        .send(process_manager::Event::new(event.clone()))
+        .await?
+    {
+        tracing::error!("Sending event to process manager failed: {:#}", e);
+    }
+    Ok(())
```

Comment:
I know you have only moved the code but failing to apply the event is a serious error and we should not return `Ok` in that case. Can be fixed in a follow-up because it is only moved here!

Reply:
ok, I'll leave a TODO now and deal with it in a follow-up!